troubleshooting.rst

```diff
@@ -51,22 +51,16 @@ No audio
 
 Updating the launcher
 ~~~~~~~~~~~~~~~~~~~~~
-Depending on your system the process may vary:
+to update the launcher on newer versions of the launcher Press the gear icon and then check for updates
 
-- Ubuntu prebuilt packages - :code:`sudo apt-get update && sudo apt-get upgrade`
+if your launcher is too old or you cant find these buttons 
+in Ubuntu prebuilt packages do sudo apt-get update && sudo apt-get upgrade 
 
-- Mac OS - Redownload the package
+and on Mac OS - Redownload the package
 
-macOS X Catalina - :code:`Could not find the game launcher..`
+macOS X Catalina
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Currently, macOS Catalina does not support 32-bit applications at this time; you will not be able to use this launcher.
```

Comment:
This mostly didn't changed with ng, but 
"you will not be able to use this launcher."
should be replaced by "you won't be able to run older Minecraft < 1.14.60.5"

Reply:
I agree.